project-jee8-webapp/src/main/java/fr/pantheonsorbonne/ufr27/miage/jms/conf/BulletinTopicSupplier.java

```diff
@@ -0,0 +1,46 @@
+package fr.pantheonsorbonne.ufr27.miage.jms.conf;
+
+import java.util.Hashtable;
+import java.util.function.Supplier;
+
+import javax.jms.Queue;
+import javax.jms.Topic;
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+
+import org.apache.activemq.artemis.jndi.ActiveMQInitialContextFactory;
+
+public class BulletinTopicSupplier implements Supplier<Topic>{
```

Comment:
That's the class supplying the bulletin topic, added based on the model of the other suppliers